### PR TITLE
Support monitoring MySQL slow sql

### LINKED ISSUE #9548: [Feature] Monitoring MySQL's Slow SQL
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

MySQL supports recording the slow SQL statement to the log file, we can monitor this file, so that we can analyze the slow query situation of MySQL.

### Step

1.Extend LAL to convert LogData into DatabaseSlowStatement for subsequent analysis.
2.Use fluent-bit to monitor MySQL's log file, and send data to OAP via http.
3.Edit the UI to display the data.
4.Add showcase and E2E test.

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yswdqz**
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [x] If it's UI related, attach the screenshots below.

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #9548 .
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

}~I7B9`V{NT3EFT%8{AFD Y

12`~4{LO}~M US4)6DS0M(0

### CODE REVIEW

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json [hunk 1]
```diff
@@ -17,317 +17,345 @@
...
+                           "y":0,
+                           "w":24,
+                           "h":29,
+                           "i":"0",
+                           "type":"Widget",
+                           "metricTypes":[
+                              "readSampledRecords"
+                           ],
+                           "metrics":[
+                              "top_n_database_statement"
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json:339]: Do we support to set N for sampled record on UI?
As you use a whole page to show. Top 50 seems better.

Comment (yswdqz) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json:339]: > Do we support to set N for sampled record on UI? As you use a whole page to show. Top 50 seems better.

We support that, and default is 10.
> As you use a whole page to show. Top 50 seems better.
I get it.

File: docs/en/setup/backend/backend-mysql-monitoring.md [hunk 2]
```diff
@@ -1,21 +1,34 @@
 # MySQL monitoring
-SkyWalking leverages prometheus/mysqld_exporter for collecting metrics data from MySQL. It leverages OpenTelemetry Collector to transfer the metrics to
+SkyWalking leverages prometheus/mysqld_exporter for collecting metrics data. It leverages OpenTelemetry Collector to transfer the metrics to
 [OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+SkyWalking also leverages fluentbit(or other log collector) for collecting slow sql statement from MySQL.
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-mysql-monitoring.md:4]:
```suggestion
SkyWalking also leverages fluentbit(or other log collector) for collecting slow sql statements from MySQL.
```

File: docs/en/setup/backend/backend-mysql-monitoring.md [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
 
 ## Data flow
+- `prometheus/mysqld_exporter`
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-mysql-monitoring.md:7]:
```suggestion
- MySQL server performance from `prometheus/mysqld_exporter`
```

File: docs/en/setup/backend/backend-mysql-monitoring.md [hunk 4]
```diff
@@ -1,21 +1,34 @@
...
 1. mysqld_exporter collect metrics data from MySQL.
 2. OpenTelemetry Collector fetches metrics from mysqld_exporter via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus gRPC Exporter or OpenTelemetry gRPC exporter.
 3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results.
 
+- `slowsql`
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-mysql-monitoring.md:12]:
```suggestion
- Collect sampled slow SQLs
```

**Review by wu-sheng — CHANGES_REQUESTED** [review]
As slow SQLs collecting has different data path from existing metrics collecting, it is better to put docs as two parts. Current style is not clear as they are mixed always.

File: docs/en/setup/backend/backend-mysql-monitoring.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
-# MySQL monitoring
-SkyWalking leverages prometheus/mysqld_exporter for collecting metrics data from MySQL. It leverages OpenTelemetry Collector to transfer the metrics to
+# MySQL server performance from `prometheus/mysqld_exporter`
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-mysql-monitoring.md:1]: Don't change level one in the docs

Comment (wu-sheng) [reply docs/en/setup/backend/backend-mysql-monitoring.md:1]: Add a level two menu for this please

**Comment by yswdqz** [comment]
Just like this?

**Comment by wu-sheng** [comment]
I reworded the docs to make things more clear. In the docs, we should use colloquial sentences, instead, formal words are better.

**Review by wu-sheng — APPROVED** [review]
LGTM. Feel free to begin the showcase this time.

**Pull request merged**